tests/test_settings.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+
+@pytest.mark.parametrize('can_toggle',
+    ['update_dragged_clients']
+)
+def test_toggle_numeric_settings(hlwm, can_toggle):
```

Comment:
The test case name should probably be something like `test_toggle_bool_setting` now

Reply:
(similarly for its counterpart)